src/pages/airport/[icao].vue

```diff
@@ -174,6 +174,50 @@
                             <div class="airport__aircraft-title_text">
                                 {{ column.title }}
                             </div>
+                            <div v-if="column.key === 'arrivals'">
+                                <div
+                                    class="airport__aircraft-title_interval"
+                                >
+                                    <div
+                                        class="airport__aircraft-title_interval_container"
+                                        :class="{ 'airport__aircraft-title_interval_container--0': store.theme === 'default' }"
+                                    >
+                                        <clock-0-to-15-icon class="airport__aircraft-title_interval_container_icon"/>
+                                        <div
+                                            class="airport__aircraft-title_interval_container_text"
+                                        >
+                                            {{ arrivalRate[0].length }}
+                                        </div>
+                                    </div>
+                                    <div
+                                        class="airport__aircraft-title_interval_container"
+                                        :class="{ 'airport__aircraft-title_interval_container--1': store.theme === 'default' }"
```

Comment:
Same comment with i in 4

Since this code is duplicated, I suggest to move it to a component

Reply:
Not sure what you mean by duplicate code. Do you mean the code in MapAirportPopup?

Reply:
Affirm

https://github.com/VATSIM-Radar/vatsim-radar/pull/239/files#diff-2fdaecb70a35e199c6607bd95310a8e8cf6c52a7acbf154810436a2741f3f1d6R151